location_names:
# - "on the floor"
- "on the table"
# - "on the tray"
# - "in the container"
- "in the bin"
# - "in the drawer"
object_names:
# - "orange_juice"
# - "red_wine"
# - "milk"
# - "iced_tea"
# - "cola"
# - "tropical_juice"
# - "juice_pack"
- "apple"
# - "pear"
- "lemon"
# - "peach"
- "banana"
# - "strawberry"
# - "orange"
# - "plum"
# - "cheezit"
# - "cornflakes"
# - "pringles"
# - "tuna"
# - "sugar"
# - "strawberry_jello"
# - "tomato_soup"
- "mustard"
# - "chocolate_jello"
- "spam"
# - "coffee_grounds"
# - "plate"
# - "fork"
# - "spoon"
# - "cup"
# - "knife"
# - "bowl"
# - "rubiks_cube"
# - "soccer_ball"
# - "dice"
# - "tennis_ball"
# - "baseball"
# - "cleanser"
# - "sponge"
